### PR TITLE
fix: Fix some errs in devops inclue role and user

### PR DESCRIPTION
**Pull request by harrisonliu5**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix some console errs and add cluster which the project and the devops in the role table.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:
```docs

```

### CODE REVIEW

File: src/actions/role.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export default {
         },
         modal: RoleEditModal,
         store,
-        module,
+        module: store.module,
```
Comment (leoendless) [inline src/actions/role.js:96]: the module has been assigned with store.module on the top

File: src/components/Base/Modal/modal.jsx [hunk 2]
```diff
@@ -105,7 +105,6 @@ export default class Modal extends React.Component {
     const modalWrapper = document.createElement('div')
     document.body.appendChild(modalWrapper)
     document.activeElement.blur()
-
```
Comment (leoendless) [inline src/components/Base/Modal/modal.jsx:108]: Unnecessary changes

File: src/components/Modals/EditAuthorization/index.jsx [hunk 3]
```diff
@@ -52,7 +52,10 @@ export default class EditAuthorizationModal extends React.Component {
 
     const roleTemplatesMap = keyBy(this.props.roleTemplates, 'name')
     const roleTemplates = props.formTemplate.roleTemplates || []
-    const roleModules = cloneDeep(ROLE_MODULES[props.module]).map(item => ({
+    const module =
+      typeof props.module === 'string' ? props.module : props.store.module
+
```
Comment (leoendless) [inline src/components/Modals/EditAuthorization/index.jsx:57]: Unnecessary changes

File: src/stores/role.js [hunk 4]
```diff
@@ -132,7 +132,7 @@ export default class RoleStore extends Base {
 
     const result = await request.get(
       `kapis/iam.kubesphere.io/v1alpha2${this.getPath(params)}/${
-        this.module
+        params.module ? params.module : this.module
```
Comment (leoendless) [inline src/stores/role.js:135]: What does this mean

File: src/stores/user.js [hunk 5]
```diff
@@ -41,14 +41,16 @@ export default class UsersStore extends Base {
...
+
     if (devops) {
       path += `/devops/${devops}`
+    } else {
+      if (workspace) {
+        path += `/workspaces/${workspace}`
+      }
+      if (namespace) {
+        path += `/namespaces/${namespace}`
+      }
```
Comment (leoendless) [inline src/stores/user.js:53]: What do these code changes mean

File: src/components/Base/Modal/modal.jsx (hunk 2 above)
Comment (harrisonliu5) [reply src/components/Base/Modal/modal.jsx:108]: this is not my changed.

Comment (leoendless) [reply src/components/Base/Modal/modal.jsx:108]: you can rebase the dev branch before you push

File: src/components/Modals/EditAuthorization/index.jsx (hunk 3 above)
Comment (harrisonliu5) [reply src/components/Modals/EditAuthorization/index.jsx:57]: which use this component, the module is not the props.module. I found props have anothe module instance of it.

File: src/stores/role.js (hunk 4 above)
Comment (harrisonliu5) [reply src/stores/role.js:135]: In the devops Role need get other module's data, but can't change current module,so add it.

File: src/stores/user.js (hunk 5 above)
Comment (harrisonliu5) [reply src/stores/user.js:53]: In the devops member, maybe only need devops keyword to request, but it has workspace or namespace keyword, it may make error request URL. so change it.

File: src/actions/role.js (hunk 1 above)
Comment (harrisonliu5) [reply src/actions/role.js:96]: when I debug it, the module has changed to other data, so change it.
